bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/LegacyBankFactory.java

```diff
@@ -258,8 +259,8 @@ public static Bank fromBanktypeId(int id, Context context) throws BankException
         }
     }
 
-    public static ArrayList<Bank> listBanks(Context context) {
-        ArrayList<Bank> banks = new ArrayList<Bank>();
+    public static List<Bank> listBanks(Context context) {
```

Comment:
Because I want to use interfaces and not concrete implementations as much as possible.